### DISCUSSION TITLE
Release v1.3.3 / 5.58.3

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#2223] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

This build fixes various issues and adds various minor improvements.

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

#### If you have issues with an update installation, just uninstall the previous version keeping the sandboxie.ini and reinstall the new build.

# Changelog

### Added
- added domain\user notation when the LogFile registry setting is applied as workaround for [#2207](https://github.com/sandboxie-plus/Sandboxie/issues/2207)
-- usage: in "HKLM\SYSTEM\CurrentControlSet\Services\SbieSvc" add REG_SZ "LogFile" with "3;[path]\Sandboxie.log"
- added option to block host processes from accessing sandboxed ones [#2132](https://github.com/sandboxie-plus/Sandboxie/issues/2132)
-- usage: DenyHostAccess=Program.exe,y
-- note: by default, this protection only applies for write access, that is, unsandboxed processes will still be able to obtain read-only access
-- to prevent host processes from obtaining read access, ConfidentialBox=y must also be set, which require a supporter certificate
- added compatibility template for ReHIPS
- added creation of all default folders in privacy box type [#2218](https://github.com/sandboxie-plus/Sandboxie/issues/2218)

### Changed
- improved SandMan settings behaviour for non admin users [#2123](https://github.com/sandboxie-plus/Sandboxie/issues/2123)

### Fixed
- fixed issues with group moving via drag and drop
- approved more required syscalls [#2190](https://github.com/sandboxie-plus/Sandboxie/issues/2190)
- fixed issues when deleting box content and the file panel view is open
- fixed issue with config protection [#2206](https://github.com/sandboxie-plus/Sandboxie/issues/2206)
- fixed issue with default box [#2195](https://github.com/sandboxie-plus/Sandboxie/issues/2195)
- fixed issue with keyboard delete shortcut for process termination

### Removed
- removed obsolete Online Armor template

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.3.3'>Release v1.3.3 / 5.58.3</a>.</em>

**Comment by erasmo52** [comment]
Thanks for your job David.
I installed Sandboxie Classic 5.58.3 and always it is not possible to execute last Skype release (8.88.0.401). 
The Workaround: UseElectronWorkaround=Skype.exe,n that isaak654 given me in the #2217 relatively to the Sandboxie Classic 5.58.2 that I signaled in #2194 doesn't work more.
Then here we are again: 
Skype version 8.88.0.401 doesn't run in Sandboxie Classic 5.58.3
Thanks for your attention

**Reply by isaak654** [reply]
The workaround is still running fine for me on this release.

**Reply by erasmo52** [reply]
I am sorru Isaak, I checked again and finally it worked fine for me too.
Thank you for your attention and forgive me for the wasted time.

**Comment by BilleBarrett** [comment]
I uninstalled Sandboxie-Plus, Microsoft did it patch Tuesday thing 
Ran both of my browsers with no problem
Reinstalled Sandboxie-Plus 1.3.3, just after opening up Chrome or Microsoft Edge the just rebooted itself
I could read 1 error "ATTEMPT TO SWITCH FROM DPC"
Then my system proceeded to just reboot with no crash errors, remove Sandboxie-Plus and I'm running OK
I'm considering creating a new Windows 11 Pro, but I discovered that this scenario happens on another computer in my system running windows 11 Pro.

**Reply by DavidXanatos** [reply]
could you please provide an appropriate kernel dump in order to debug this "ATTEMPT TO SWITCH FROM DPC" issue
also for testing please disable network firewall tracing that is remove all NetFwTrace= from your sandboxie.ini
Alternatively you could disable WFP support.

I would really like to solve this issue as it seams to plage you for some builds already... but to do that I need some cooperation from your side.

**Comment by BilleBarrett** [comment]
I just installed Sandboxie-Plus 1.3.3 and followed your instructions

disable network firewall tracing that is remove all NetFwTrace= from your sandboxie.ini

Alternatively, you could disable WFP support.

16:37 send 1st and then open Chrome





  <https://go.sentinelone.com/rs/327-MNM-087/images/nurture.bannerv2.1200x200(1).jpg> 
End Point Detection & Response System
Remote Desktop Services
517 Miltwood Dr,  Greensboro, NC 27455-2709
(336) 988-0891    ***@***.***> ***@***.*** 
To subscribe to my FYI emails send an email to
 ***@***.***> ***@***.***
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’

**Comment by BilleBarrett** [comment]
Just started Sandboxie=Plus 1.3.3 Chrome got the following message SBIE1242 Monitor buffer overflow

 

  <https://go.sentinelone.com/rs/327-MNM-087/images/nurture.bannerv2.1200x200(1).jpg> 
End Point Detection & Response System
Remote Desktop Services
517 Miltwood Dr,  Greensboro, NC 27455-2709
(336) 988-0891    ***@***.***> ***@***.*** 
To subscribe to my FYI emails send an email to
 ***@***.***> ***@***.***
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’

**Reply by DavidXanatos** [reply]
Monitor buffer overflow, then disable some tracing messages seams you have to much enabled

**Comment by BilleBarrett** [comment]
Got these messages:

SBIE1242 Monitor buffer overflow

Chrome.exe (9576); SBIE2203 Failed to communicate with Sandboxie service connect c00000022

Chrome.exe (9576); SBIE2203 Failed to communicate with Sandboxie "GUIPPTOXY_00000001; Mysid 14 chrome.exe [C00000080]Chrome.exe (9576); SBIE2203 Failed to communicate with Sandboxie "GUIPPTOXY_00000001; Msgid 14 chrome.exe [C00000080]

Chrome.exe (9576); SBIE2203 Failed to communicate with Sandboxie "GUIPPTOXY_00000001; Msgid 14 chrome.exe [C00000080]

Chrome.exe (9576); SBIE2203 Failed to communicate with Sandboxie "GUIPPTOXY_00000001; Msgid 14 chrome.exe [C00000080]

Chrome.exe (9576); SBIE2203 Failed to communicate with Sandboxie "GUIPPTOXY_00000001; Msgid 14 chrome.exe [C00000080]

Chrome.exe (9576); SBIE2203 Failed to communicate with Sandboxie "GUIPPTOXY_00000001; Msgid 14 chrome.exe [C00000080]

 

Still running 

 

  <https://go.sentinelone.com/rs/327-MNM-087/images/nurture.bannerv2.1200x200(1).jpg> 
End Point Detection & Response System
Remote Desktop Services
517 Miltwood Dr,  Greensboro, NC 27455-2709
(336) 988-0891    ***@***.***> ***@***.*** 
To subscribe to my FYI emails send an email to
 ***@***.***> ***@***.***
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’

**Comment by BilleBarrett** [comment]
IT now almost 9 pm and my system has not rebooted 

 

  <https://go.sentinelone.com/rs/327-MNM-087/images/nurture.bannerv2.1200x200(1).jpg> 
End Point Detection & Response System
Remote Desktop Services
517 Miltwood Dr,  Greensboro, NC 27455-2709
(336) 988-0891    ***@***.***> ***@***.*** 
To subscribe to my FYI emails send an email to
 ***@***.***> ***@***.***
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’

**Comment by BilleBarrett** [comment]
Do I have to edit NetFwTrace= from your sandboxie.ini or is there a setting that need to be check or unchecked

 


517 Miltwood Dr,  Greensboro, NC 27455-2709
(336) 988-0891    ***@***.***> ***@***.*** 
To subscribe to my FYI emails send an email to
 ***@***.***> ***@***.***
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’

**Comment by BilleBarrett** [comment]
I also, that was what I saw when it blue screened, sometime it just plain
reboots.
I am considering install my windows 11 pro again and reinstall all my
software to remove that variable.
I have run hardware diags but nothing failed

**Comment by BilleBarrett** [comment]
Should tracing be turned off completely, I was under the impression it was a tool to help you see what is going on with Sandboxie-Plus

Bill

  <https://go.sentinelone.com/rs/327-MNM-087/images/nurture.bannerv2.1200x200(1).jpg> 
End Point Detection & Response System
Remote Desktop Services
517 Miltwood Dr,  Greensboro, NC 27455-2709
(336) 988-0891    ***@***.***> ***@***.*** 
To subscribe to my FYI emails send an email to
 ***@***.***> ***@***.***
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’
